l10n_it_statutory_financial_statements_mapping/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Francesco Apruzzese
+# Copyright 2019 Ilaria Franchini
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Statutory_financial statements mapping",
```

Comment:
Dato che non è traducibile in weblate, il nome del modulo andrebbe in italiano, come da linee guida

https://www.odoo-italia.org/documentazione/sviluppo/guidelines.html

Se riesci sarebbe forse utile anche un summary (sempre in italiano).